### DISCUSSION TITLE
Mac OS

### DISCUSSION THREAD

**Discussion by TomClarkMassSpec** [levitsky/pyteomics#29] [Q&A] [answered]
Hello!
The documents on package installation (pyteomics for use with Python) does not mention installation for Mac OS such as Big Sur. Is this not going to work, or just not tested?
Thanks,
Tom

**Comment by TomClarkMassSpec** [comment, marked as answer]
Update my own question here: enter the following into terminal. 
conda install -c bioconda pyteomics
All appears well in Big Sur Mac OS/CONDA with Jupiter notebook and Spyder.

**Reply by levitsky** [reply]
Thank you Tom!

Indeed, installing through `conda` or `pip` should work just fine, but I never had the chance to test it.
